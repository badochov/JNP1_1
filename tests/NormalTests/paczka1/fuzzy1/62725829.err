Error in line 9: PCV A66 24,6
Error in line 2: A66 S396 12,5
Error in line 12: PCV A847 18,5
Error in line 10: sVUghEjxSAX S396 16,2
Error in line 15: E2FMmMI A263 79,9
Error in line 35: EfFFZLBb5iV A66 7|,5
Error in line 34: 6N5 A66 56,8
