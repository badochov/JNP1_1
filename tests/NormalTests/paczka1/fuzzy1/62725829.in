sVUghEjxSAX A263 81,1
A66 S396 12,5
PCV A847 89,3
PCV A847 295,6
?iB08JYTy
EfFFZLBb5iV S396 79,8
EfFFZLBb5iV S396 160,4
sVUghEjxSAX A263 145,2
PCV A66 24,6
sVUghEjxSAX S396 16,2
?iB08JYTy
PCV A847 18,5
iB08JYTy S396 13,5
A66 A263 37,8
E2FMmMI A263 79,9
PCV A66 72,1

sVUghEjxSAX A66 15,5
E2FMmMI A847 10,5
?iB08JYTy
sVUghEjxSAX A66 66,8
6N5 A263 89,5
sVUghEjxSAX A66 56,8
PCV A66 241,5
iB08JYTy S396 16,6
6N5 A263 250,6
E2FMmMI A847 33,9
A66 A263 131,2
E2FMmMI A263 44,7
E2FMmMI A263 192,2
EfFFZLBb5iV S396 43,2
iB08JYTy A66 74,8
E2FMmMI A66 45,6
6N5 A66 56,8
EfFFZLBb5iV A66 7|,5
iB08JYTy A66 112,9
A66 S396 76,1
?
6N5 A263 8,7
E2FMmMI A66 199,7