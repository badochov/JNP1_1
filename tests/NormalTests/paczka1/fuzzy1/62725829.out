6N5 A 161,1
A66 A 93,4
E2FMmMI A 170,9
EfFFZLBb5iV S 80,6
PCV A 375,7
iB08JYTy A 38,1 S 3,1
sVUghEjxSAX A 115,4
A66 258,8
A263 466,1
S396 83,7
A847 229,7
